src/main/java/com/example/demo/service/MovieService.java

```diff
@@ -0,0 +1,92 @@
+package com.example.demo.service;
+
+import com.example.demo.entity.Booked;
+import com.example.demo.entity.BookedSeat;
+import com.example.demo.entity.Movie;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+@Service
+public class MovieService {
+    public Booked seatReservation(Movie movie, BookedSeat[] seatsNr) {
+
+        BigDecimal ticketPrice = BigDecimal.ZERO;
+        Boolean[] seats = movie.getSeats();
+        for(BookedSeat nr : seatsNr) {
+            if (seats[nr.getSeatNr()].equals(Boolean.FALSE)) {
+                seats[nr.getSeatNr()] = Boolean.TRUE;
+                movie.setSeats(seats);
+                int x = nr.getSeatNr() / movie.getTotalColumns();
```

Comment:
czy `x` to dobra nazwa?

Reply:
Powinienem nadać bardziej znaczącą nazwę typu `nrOfSeatsPerColumns`